samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -176,6 +176,7 @@ const StorybookDecorator = (story, config) => {
 	const currentSkin = skinFromURL ? skinFromURL : Config.defaultProps.skin;
 	const newSkin = (memory.skin !== currentSkin);
 	memory.skin = currentSkin;  // Remember the skin for the next time we load.
+	const useSkinDefaultStyles = boolean('default skin styles', Config);
```

Comment:
It looks better to re-order knobs so the new knob could be the next position of `skin` knob.

Reply:
BTW, a name starting with `use` is okay? :)